isis/src/lro/apps/lronac2isis/LroNacArchive.trn

```diff
@@ -0,0 +1,250 @@
+#
```

Comment:
Do you mean to have LRO stuff in here?

Reply:
No, this one slipped into this PR some how. It needs to be in the branch, so I'll leave it for now.

Reply:
No, I need to remove the LRO files